ecosystem/sep-0006.md

```diff
@@ -33,11 +34,14 @@ To support this protocol an anchor acts as a server and implements the specified
 
 * An anchor must define the location of their `TRANSFER_SERVER` in their [`stellar.toml`](sep-0001.md). This is how a wallet knows where to find the anchor's server.
 * Anchors and clients may support [SEP-10](sep-0010.md) web authentication to enable authenticated deposits, withdrawals, or transaction history lookups.
+* If the Anchor supports [SEP-38], `ANCHOR_QUOTE_SERVER` must be defined in their [`stellar.toml`](sep-0001.md).
```

Comment:
❔ SEP-38 doesn't mention this stellar.toml field. Should we mention it there instead or as well?

Reply:
That's a good point! It's currently being covered only in [SEP-1](https://github.com/stellar/stellar-protocol/blame/926cf0dfb9d4b8470efc7b354e2a5865892210c4/ecosystem/sep-0001.md#L64). I will add that to SEP-38 in another PR.

Reply:
I added it to SEP-38 in https://github.com/stellar/stellar-protocol/pull/989/commits/bad6e317a04d2f51092a87213508814bf96f9a85.

I think it's ok to keep it in SEP-6 as well.